model/src/application/interaction/mod.rs

```diff
@@ -521,7 +521,7 @@ mod test {
                 Token::Str("name"),
                 Token::Str("member"),
                 Token::Str("type"),
-                Token::U8(CommandOptionType::User as u8),
+                Token::U8(6),
```

Comment:
I wonder if it's not best to keep these as-is (using `u8::from`) for readability and writability of tests and just have tests for the variant numerical values at the source.